spec/factories/rewards.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :reward do
+    title { Faker::Lorem.word }
+    description { Faker::Quotes::Shakespeare.hamlet_quote }
+    price { rand(1000) }
```

Comment:
this may result with 0, and you will get an unexpected error in the spec :) 
```suggestion
    price { rand(1..1000) }
```